### PR TITLE
Add recently modified/failed jobs to the report

### PR DESCRIPTION
**Pull request by bradyoo**
Don't need this in 2.1.0 so it can wait

Examples:

example_01
example_02

### CODE REVIEW

**Comment by bradyoo** [comment]
@calvinjia

**Comment by calvinjia** [comment]
@bradyoo Nice! Is it a large change to add the job type to `JobInfo`?

**Comment by bradyoo** [comment]
I was going to look at some other things first. And if we wanted this for 2.1.0, we should put this in as is.
 
The things I wanted to look at before are:
1. generateTarballs taking significantly longer
2. I'm noticing that the master is having these phantom jobs that are not actually real. You can see above that it claims it has 48 running but I doubt they are actually running anywhere.

File: shell/src/main/java/alluxio/cli/fsadmin/report/JobServiceMetricsCommand.java [hunk 1]
```diff
@@ -52,6 +59,31 @@ public int run() throws IOException {
...
+
+    for (JobInfo lastActivity : lastActivities) {
+      mPrintStream.print(String.format("Timestamp: %-30s",
+          CommonUtils.convertMsToDate(lastActivity.getLastStatusChangeMs(), mDateFormatPattern)));
+      mPrintStream.print(String.format("Job Id: %-20s", lastActivity.getJobId()));
+      mPrintStream.println(String.format("Status: %s", lastActivity.getStatus()));
+    }
+
+    mPrintStream.println();
+    mPrintStream.println("Last 10 Failures:");
```
Comment (calvinjia) [inline shell/src/main/java/alluxio/cli/fsadmin/report/JobServiceMetricsCommand.java]: This is not 100% accurate right, its more like `last 10 failures in recent history`?

Comment (bradyoo) [reply shell/src/main/java/alluxio/cli/fsadmin/report/JobServiceMetricsCommand.java]: Yeah I should say Last 10 **** in recent history for both.

**Comment by bradyoo** [comment]
Addressed feedback + fixed test.

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Pull request merged**